### PR TITLE
Prepare 7.24.0

### PR DESCRIPTION
**Pull request by JimiC**
**Changes proposed:**

- [x] Prepare

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
 # Change Log
 
+## 7.24.0 (June 19, 2018)
+
+- Enhancement: Associated `plist` with `config` icon. ([@JimiC](https://github.com/JimiC) in [#1630](https://github.com/vscode-icons/vscode-icons/pull/1630))
+- Misc: Disable `before_script` step in docker stages. ([@JimiC](https://github.com/JimiC) in [#1629](https://github.com/vscode-icons/vscode-icons/pull/1629))
```
Comment (robertohuertasm) [inline CHANGELOG.md]: Until now we've used `Feature`, `Fix` and `Enhancement` for everything. I think that we can put all those `Misc` logs into one of those three categories.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+- Enhancement: Added support for `Django` via language Id. ([@batisteo](https://github.com/batisteo) in [#1622](https://github.com/vscode-icons/vscode-icons/pull/1622))
+- Enhancement: Added support for `.babelrc.js` and `.babelignore`. ([@zalishchuk](https://github.com/zalishchuk) in [#1621](https://github.com/vscode-icons/vscode-icons/pull/1621))
+- Misc: Updated examples generator dependency. ([@JimiC](https://github.com/JimiC) in [#1618](https://github.com/vscode-icons/vscode-icons/pull/1618))
+- Misc: Moved `mocha` options to an options file. ([@JimiC](https://github.com/JimiC) in [#1617](https://github.com/vscode-icons/vscode-icons/pull/1617))
+- Misc: Removed `bithound` config file. ([@JimiC](https://github.com/JimiC) in [#1609](https://github.com/vscode-icons/vscode-icons/pull/1609))
+- Enhancement: Added support for `Jenkins` via language Id. ([@otorrillas](https://github.com/otorrillas) in [#1607](https://github.com/vscode-icons/vscode-icons/pull/1607))
+- Misc: Improved `AppVeyor` config. ([@JimiC](https://github.com/JimiC) in [#1603](https://github.com/vscode-icons/vscode-icons/pull/1603))
```
Comment (robertohuertasm) [inline CHANGELOG.md]: For instance, this could be an `Enhancement`.

**Comment by JimiC** [comment]
@robertohuertasm Can you please be so kind, and do the changes as well as commit them, so we can proceed with the release?

**Comment by robertohuertasm** [comment]
@JimiC, all ready to tag.The honour is all yours!

**Comment by robertohuertasm** [comment]
@JimiC superb work with the publication flow!!

**Pull request merged**